src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -303,17 +303,18 @@ function AddTaskModal(props) {
           </button>
         </ModalHeader>
         <ModalBody>
-          <table className="table table-bordered responsive">
-            <tbody>
-              <tr>
-                <td scope="col" data-tip="WBS ID">
+          <div className="table table-bordered responsive">
+            <div>
+              <div className="m-0 border d-flex align-items-center">
+                <span className="p-1 " data-tip="WBS ID" style={{width: "30%"}}>
                   WBS #
-                </td>
-                <td scope="col">{newTaskNum}</td>
-              </tr>
-              <tr>
-                <td scope="col">Task Name</td>
-                <td scope="col">
+                </span>
```

Comment:
adding in line style it can be cumbersome when it comes to refractor the code. You repeating the inline style multiple times. You could simply create a class on the css file and add under className. That way is easier to refractor and maintain the code.

Reply:
Hello @mSharifHub , thank you so much for reviewing this PR and great suggestion. Will try to implement the changes to make the code more maintainable.